reporter/src/main/kotlin/reporters/ReportTableModel.kt

```diff
@@ -80,6 +80,12 @@ data class ReportTableModel(
              */
             val rows: List<DependencyRow>,
 
+            /**
+             * The path to the directory containing the definition file of the project, relative to the analyzer root,
+             * see [OrtResult.getDefinitionFilePathRelativeToAnalyzerRoot].
+             */
+            val fullDefinitionFilePath: String,
```

Comment:
For the record, I still don't like "full" here for the same reason I didn't like "complete": Both sound as if nothing was missing from the path, but as it still is a relative path a prefix *is* missing. Anyway, I'll stop nit-picking and approve 😉

Reply:
I'll change it in a separate PR.